src/hooks/useFeedbackMessage/useFeedbackMessage.module.css

```diff
@@ -50,10 +50,18 @@
   }
 
   [class~=ant-message-notice-content] {
+    align-items: center;
     padding: calc(var(--spacing-padding-md, 8) * 1px) !important;
+    display: inline-flex !important;
+    height: 40px;
 
     [class~=ant-message-custom-content] {
       display: flex !important;
+      align-items: center;
+
+      [class~=anticon-loading] {

```

Comment:
We should try to achieve the same behavior using Ant Design Tokens instead.
https://ant.design/components/message#design-token

Reply:
we tried but it's easier this way